v2.6/reference/kube-controllers/configuration.md

```diff
@@ -1,17 +1,24 @@
 ---
-title: Configuring the Calico policy controller
+title: Configuring the Calico Kubernetes controllers
 ---
 
-The policy controller is primarily configured through environment variables. When running
-the policy controller as a Kubernetes pod, this is accomplished through the pod manifest `env`
+The Kubernetes controllers are primarily configured through environment variables. When running
+the controllers as a Kubernetes pod, this is accomplished through the pod manifest `env`
 section.
 
-## The calico/kube-policy-controller container
+## The calico/kube-controllers container
+
+The calico/kube-controllers container includes three controllers:
+
+1) policy controller: watches network policies and programs Calico policies.
```

Comment:
surprised to see github rendering this fine, didn't know that an end parenthesis is accepted. Thought you'd need:
```
1. policy controller...
2. profile controller...
3. wep controller...
```
or even 
```
1. policy controller...
1. profile controller...
1. wep controller...
```
@emanic are we standardizing on this or is this OK?

Reply:
Yes thanks @ozdanborne this is incorrect.................
Because the sequence of these items does not matter, it should be a bulleted list, not a numbered list. But, if it was a numbered list, you should use a period after the number. This allows GitHub to keep track of the numbers for you (a la HTML lists) and you don't have to manually manage the numbers.